src/cdk/text-field/autosize.ts

```diff
@@ -37,10 +39,12 @@ import {Subject} from 'rxjs/Subject';
 export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
   /** Keep track of the previous textarea value to avoid resizing when the value hasn't changed. */
   private _previousValue: string;
+  private _previousHeight = 0;
```

Comment:
why don't we do `_previousHeight: number;` and then initialize it at the same time that we call `_cacheTextareaLineHeight`. That way it won't fire an event the first time the user types unless the height actually changes.

Reply:
I wanted it to fire initially though. But given it's `sizeChanged` it makes sense for it not to fire.